.gitpod.yml

```
@@ -0,0 +1,12 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - init: |
+      pip install mkdocs-material 
+      pip install mkdocs-material-extensions mkdocs-macros-plugin mkdocs-exclude mkdocs-awesome-pages-plugin mkdocs-redirects # runs during prebuild
+    command: mkdocs serve
```

```suggestion
      pip install -r requirements.txt
    command: mkdocs serve --dirtyreload
```
This will allow to use same versions of dependencies we are using in netlify and also make livereload faster